### PR TITLE
Corrected "on-prem" typo to "on-premises"

### PR DESCRIPTION
**Pull request by aj11anuj**
Fixed a typo "on-prem" to "on-premises"

### CODE REVIEW

**Comment by aj11anuj** [comment]
I don't know what's the dco problem... i have used "-S" while commiting and also added that signed of message still this issue persists

**Comment by kevburnsjr** [comment]
> I don't know what's the dco problem... i have used "-S" while commiting and also added that signed of message still this issue persists

image
image

File: website/docs/intro/index.mdx [hunk 1]
```diff
@@ -6,9 +6,9 @@ description: |-
 
 # What is OpenTofu?
 
-OpenTofu is an infrastructure as code tool that lets you define both cloud and on-prem resources in human-readable configuration files that you can version, reuse, and share. You can then use a consistent workflow to provision and manage all of your infrastructure throughout its lifecycle. OpenTofu can manage low-level components like compute, storage, and networking resources, as well as high-level components like DNS entries and SaaS features.
+OpenTofu is an infrastructure as code tool that lets you define both cloud and on-premises resources in human-readable configuration files that you can version, reuse, and share. You can then use a consistent workflow to provision and manage all of your infrastructure throughout its lifecycle. OpenTofu can manage low-level components like compute, storage, and networking resources, as well as high-level components like DNS entries and SaaS features.
 
-## How does OpenTofu work?
+## How does OpenTofu works?
```
Comment (Hefeweizen) [inline website/docs/intro/index.mdx:11]: The original form was grammatically correct for English.  Consider reverting this portion of the change.

**Comment by marcinwyszynski** [comment]
Isn't `on-prem` a valid word? I used it a lot myself, for brevity.

**Comment by serdardalgic** [comment]
`on-prem` looks like an abbreviation of `on-premises`, according to Wikipedia. 

> On-premises software (abbreviated to on-prem, and incorrectly referred to as on-premise)[[1]](https://en.wikipedia.org/wiki/On-premises_software#cite_note-1) is installed and runs on computers on the [premises](https://en.wikipedia.org/wiki/Premises) of the person or organization using the software, rather than at a remote facility such as a [server farm](https://en.wikipedia.org/wiki/Server_farm) or [cloud](https://en.wikipedia.org/wiki/Cloud_computing).

AFAIU: 
`on-prem` ✅
`on-premises` ✅ 
`on-premise` ❌ 

So, `on-prem` can stay, my 2 cents.

**Comment by eranelbaz** [comment]
> > I don't know what's the dco problem... i have used "-S" while commiting and also added that signed of message still this issue persists
> 

There are two sign methods - 
- GPG singing 
- DCO signing

The DCO signing is to sign that
- You are the only author of the commits in this branch
- You are absolutely certain nobody else is doing any work based upon this branch
- There are no empty commits in the branch (for example, a DCO Remediation Commit which was added using --allow-empty)

Plus for GPG signing you need to configure GPG key in your personal github user settings

**Comment by marcinwyszynski** [comment]
I'm going to close this one because `on-prem` is widely used as-is.

**Pull request closed without merging**